<launch> 

  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

  <node pkg="stomp_ros_interface" name="display_robot_model" type="display_robot_model" respawn="false" output="screen" clear_params="true" launch-prefix="$(arg launch_prefix)">
    <rosparam command="load" file="$(find darpa_arm_robot_arm_navigation)/config/right_arm_stomp_config.yaml" />
    <param name="group" value="R_ARM"/>
  </node>
</launch>
